chain/chain/src/tests/simple_chain.rs

```diff
@@ -39,8 +39,9 @@ fn build_chain() {
     // ‘nightly’ feature enabled:
     //
     //     cargo install cargo-insta
-    //     cargo insta test --accept -p near-chain                    -- tests::simple_chain::build_chain
-    //     cargo insta test --accept -p near-chain --features nightly -- tests::simple_chain::build_chain
+    //     cargo insta test --accept -p near-chain                    --
```

Comment:
This would probably have been handled better if it was inside a markdown code block…? Not sure.